generator-jsonforms/src/generators/app/index.ts

```diff
@@ -0,0 +1,173 @@
+// tslint:disable:no-var-requires
+// tslint:disable:no-require-imports
+'use strict';
+
+import * as Generator from 'yeoman-generator';
+import chalk from 'chalk';
+const clear = require('clear');
+const figlet = require('figlet');
+const validate = require('validate-npm-package-name');
+import { join, sep } from 'path';
+import { readFile, writeFile } from 'fs';
+
+enum ProjectRepo {
+  Example = 'make-it-happen-react',
+  Seed = 'jsonforms-react-seed',
+}
+
+enum Project {
+  Example = 'example',
+  Seed = 'seed',
+}
+
+export class JsonformsGenerator extends Generator {
+
+  project: string;
+  repo: string;
+  path: string;
+  name: string;
+  skipPromting = false;
+  answers: any;
+
+  constructor(args: any, opts: any) {
+    super(args, opts);
+
+    this.option('project', { type: String } );
+    this.option('path', { type: String } );
+    this.option('name', { type: String } );
+    this.option('skipPromting', { type: Boolean} );
+
+    this.project = this.options.project;
+    this.repo = '';
+    this.path = this.options.path;
+    this.name = this.options.name;
+    this.skipPromting = this.options.skipPromting;
+
+    if (this.project === Project.Example) {
+      this.repo = ProjectRepo.Example;
+    }
+    if (this.project === Project.Seed) {
+      this.repo = ProjectRepo.Seed;
+    }
+  }
+
+  async prompting() {
+    clear();
+    this.log(
+      chalk.blue(
+        figlet.textSync('JSONForms Tooling', { horizontalLayout: 'full' }),
+      ),
+    );
+    if (!this.skipPromting) {
+      this.answers = await this.prompt([
+        {
+          name: 'project',
+          type: 'list',
+          message: 'Select a project',
+          choices: [
+            {
+              name: 'Example Project',
+              value: ProjectRepo.Example
+            },
+            {
+              name: 'Seed Project',
+              value: ProjectRepo.Seed
+            }
+          ],
+          when: () => {
+            if (this.project == null) {
+              return true;
+            }
+            /* if (!(Object.values(Project).includes(this.project))) {
```

Comment:
Remove commented code

Reply:
This commented code refers to a question I posted in the technical-questions channel on slack.
This code should check if a value is inside a enum. But I wasn't able to get it working. Do you have any idea here?